internal/operators/ocs/ocs_operator_test.go

```diff
@@ -0,0 +1,111 @@
+package ocs_test
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/operators/ocs"
+	"github.com/openshift/assisted-service/models"
+)
+
+var _ = Describe("Ocs Operator Test GetHostRequirements", func() {
+	var (
+		ctx            = context.TODO()
+		operator       = ocs.NewOcsOperator(common.GetTestLog())
+		master1        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		master2        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		master3        = &models.Host{Role: models.HostRoleMaster, Inventory: defaultInventory()}
+		worker1        = &models.Host{Role: models.HostRoleWorker, Inventory: ocsInventoryWithTwoDisks(12, 64000000000)}
+		worker2        = &models.Host{Role: models.HostRoleWorker, Inventory: ocsInventoryWithDisks(12, 64000000000)}
+		worker3        = &models.Host{Role: models.HostRoleWorker, Inventory: ocsInventoryWithDisks(12, 64000000000)}
+		autoAssignHost = &models.Host{Role: models.HostRoleAutoAssign}
+	)
+
+	test := []struct {
```

Comment:
What do you think about using Ginkgo's tabular tests (similarly to https://github.com/openshift/assisted-service/blob/35cf3471a9f655a7fee5052aa72577cf518faa14/internal/operators/manager_test.go#L201)?

Reply:
Sure, I will use it